clients/include/testing_syr_strided_batched.hpp

```diff
@@ -75,21 +72,20 @@ void testing_syr_strided_batched(const Arguments& arg)
     // argument check before allocating invalid memory
     if(N <= 0 || lda < N || lda < 1 || !incx || batch_count <= 0)
     {
-        static constexpr size_t safe_size = 100; // arbitrarily set to 100
-
-        device_vector<T> dA_1(safe_size);
-        device_vector<T> dx(safe_size);
-        if(!dA_1 || !dx)
-        {
-            CHECK_HIP_ERROR(hipErrorOutOfMemory);
-            return;
-        }
-
-        EXPECT_ROCBLAS_STATUS(
-            rocblas_syr_strided_batched<T>(
-                handle, uplo, N, &h_alpha, dx, incx, stridex, dA_1, lda, strideA, batch_count),
-            N < 0 || lda < N || lda < 1 || !incx || batch_count < 0 ? rocblas_status_invalid_size
-                                                                    : rocblas_status_success);
+        EXPECT_ROCBLAS_STATUS(rocblas_syr_strided_batched<T>(handle,
+                                                             uplo,
+                                                             N,
+                                                             &h_alpha,
+                                                             nullptr,
```

Comment:
Interesting, so you pass `nullptr` instead of unnecessarily allocating `safe_size` when passing other quick-return success or invalid arguments, since the pointers don't need to be dereferenced if there's a quick-return success or an invalid argument, and the pointers aren't checked until after the quick-return success or other invalid argument checks.

I did not originate the `safe_size` idea; it has been there for a while. But this is much simpler.